eras/dijkstra/impl/src/Cardano/Ledger/Dijkstra/Rules/Mempool.hs

```diff
@@ -82,8 +151,8 @@ instance
   type Signal (DijkstraMEMPOOL era) = Tx TopTx era
   type Environment (DijkstraMEMPOOL era) = LedgerEnv era
   type BaseM (DijkstraMEMPOOL era) = ShelleyBase
-  type PredicateFailure (DijkstraMEMPOOL era) = DijkstraLedgerPredFailure era
-  type Event (DijkstraMEMPOOL era) = ConwayLedgerEvent era
+  type PredicateFailure (DijkstraMEMPOOL era) = DijkstraMempoolPredFailure era
+  type Event (DijkstraMEMPOOL era) = DijkstraMempoolEvent era
```

Comment:
I don't think we need to create this wrapper, since it is very unlikely that we will ever emit ledger events from the mempool itself.
Ledegr events are relevant for block validation, which MEMPOOL rule does not participate in.
```suggestion
  type Event (DijkstraMEMPOOL era) = DijkstraLedgerEvent era
```
So, let's revert the changes related to  `DijkstraMempoolEvent`.